### PR TITLE
Add just file for Ankaios

### PR DESCRIPTION
**Pull request by krucod3**
This PR adds a just file to the Ankaios repo
It's a first version of the file that at least helps with some tasks like executing stest with freshly build debug binaries.

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [x] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

**Comment by windsource** [comment]
Looks good but I have some minor comments:

1. Replace tabs by spaces in `justfile` (caused by copy-paste from `Makefile`)
2. To avoid confusing remove `Makefile`
3. In https://github.com/eclipse-ankaios/ankaios/blob/0af7daf9e058848ff1474aa6616139886324beb6/.github/workflows/build.yml#L63 and https://github.com/eclipse-ankaios/ankaios/blob/0af7daf9e058848ff1474aa6616139886324beb6/.github/workflows/build.yml#L69 use `just` instead of `make`

**Comment by krucod3** [comment]
Thanks for the review. Findings fixed.

File: justfile [hunk 1]
```diff
@@ -12,17 +12,31 @@
...
+build:
+    cargo build
+
+build-release:
+    cargo build --release
 
-all: check-test-images check-licenses
+clean:
+    cargo clean
+    ./tools/dev_scripts/ankaios-clean
```
Comment (windsource) [inline justfile:25]:
```suggestion
    ./tools/dev_scripts/ankaios-clean
    rm -rf build
```

File: justfile [hunk 2]
```diff
@@ -12,17 +12,31 @@
...
-	cargo deny check licenses
+    cargo deny check licenses
 
 # Prevent non ghcr.io images to be used in test due to rate limit problem
 check-test-images:
-	test -z "$(shell find tests/resources/configs -type f -exec grep -H -P 'image: (?!ghcr\.io/|image_typo:latest)' {} \;)"
+    test -z "$(find tests/resources/configs -type f -exec grep -H -P 'image: (?!ghcr\.io/|image_typo:latest)' {} \;)"
+
+test:
+    cargo test
```
Comment (windsource) [inline justfile:35]: In CI build we use `cargo nextest run`. Shouldn't we use the same here?

**Review by windsource — CHANGES_REQUESTED** [review]
Looks good but two minor remarks added

File: justfile (hunk 2 above)
Comment (krucod3) [reply justfile:35]:
```suggestion
    cargo nextest run
```

**Review by windsource — APPROVED** [review]
LGTM 👍

**Pull request merged**